torch/fx/experimental/symbolic_shapes.py

```diff
@@ -818,6 +818,8 @@ def _iterate_exprs(val: IterateExprs) -> Iterator[sympy.Basic]:
         yield from _iterate_exprs(val.storage_offset())
     elif val is None:
         pass
+    elif isinstance(val, torch._C.Generator):
```

Comment:
nit: can you add a comment like `Note: [Generator arguments in AOTDispatcher]` somewhere in the AOT code that you reference in any places that now need to check for `Generator` types, so people have some bread crumbs they can follow if they're wondering why they need to handle generator args?